code/scripting/api/libs/mission.cpp

```diff
@@ -1269,6 +1269,28 @@ ADE_FUNC(isInCampaign, l_Mission, NULL, "Get whether or not the current mission
 	return ade_set_args(L, "b", b);
 }
 
+ADE_FUNC(isNebula, l_Mission, nullptr, "Get whether or not the current mission being played is set in a nebula", "boolean", "true if in nebula, false if not")
+{
+	bool b = false;
+
+	if (The_mission.flags[Mission::Mission_Flags::Fullneb]) {
+		b = true;
+	}
+
+	return ade_set_args(L, "b", b);
```

Comment:
Would be easier to just write it like this:
```suggestion
	return ade_set_args(L, "b", The_mission.flags[Mission::Mission_Flags::Fullneb]);
```

Reply:
Yeah, only did it like this cause isInCampaign did it too.